.travis.yml

```diff
@@ -164,6 +163,18 @@ matrix:
       before_script:
         - psql -c 'create database sentry;' -U postgres
 
+    # django 1.8 compatibility with migrations
```

Comment:
fwiw this is basically just testing 'does syncdb work' right now, which is better than nothing, but eventually itll test actually migrations once we build those